### ISSUE TITLE
AttributeError: module 'docutils.nodes' has no attribute 'meta'

### ISSUE THREAD

**Issue by tbarbette** [ammaraskar/sphinx-action#58]
Hi all,

This action used to work but recently (after an Ubuntu update?) I got the following error (more at
https://github.com/tbarbette/npf-docs/actions/runs/8567891792/job/23480593613)

```
AttributeError: module 'docutils.nodes' has no attribute 'meta'
make: *** [Makefile:20: html] Error 1
[sphinx-action] Starting sphinx-action build.
Running: 
```

**Comment by david-lev** [comment]
@tbarbette Any solution? I have the same err when I'm building my docs in readthedocs.

This is my `requirements.txt `

```cfg
sphinx-book-theme==1.1.2
sphinx-copybutton==0.5.2
sphinxext-opengraph==0.9.1
sphinx-togglebutton==0.3.2
m2r==0.3.1
```
And `.readthedocs.yaml`

```yml
build:
  os: ubuntu-22.04
  tools:
    python: "3.11"
```
Build log: https://beta.readthedocs.org/projects/pywa/builds/24111823/

**Comment by tbarbette** [comment]
Yes, simply use "tbarbette/sphinx-action@master" instead of ammaraskar's one. The only difference is this : 

https://github.com/ammaraskar/sphinx-action/commit/24d656458771553b2b4d0d731d6e0d95f1bc3f7e

**Comment by Aschwins** [comment]
Adding a release tag also works, in your case @7.1.2. Like mentioned here #56, all releases can be found [here](https://github.com/ammaraskar/sphinx-action/tags).

```
    - name: Build HTML
      uses: ammaraskar/sphinx-action@8.1.3
```
